reference/requirements.rst

```diff
@@ -13,30 +13,30 @@ The following requirements *must* be met to run Sulu:
 * :doc:`Apache <../cookbook/web-server/apache>` or
   :doc:`Nginx <../cookbook/web-server/nginx>` with enabled URL rewriting
 * `PHP`_ 7.2 or higher
-* the `intl_extension`_ for PHP
-* the `fileinfo` extension for PHP
+* the `json_extension`_, `dom_extension`_ and `xml_extension`_ for PHP
```

Comment:
since PHP 8 json extension is a core extension so we can remove it on the `2.5` branch where we also will increase the PHP requirement to 8.1